src/main/java/org/beta/authentification/keycloak/bot/BotSenderFactory.java

```diff
@@ -9,9 +9,9 @@
 public class BotSenderFactory {
     private static BotSender instance;
 
-    public static BotSender getInstance() {
+    public static BotSender getInstance(String email, String password) {
```

Comment:
BotSender is a singleton, would it work as expected? Each client should have a different botSender as email/password are per client.

Reply:
It's working when the singleton is commented.